### PR TITLE
Homework submission

### PR DESCRIPTION
**Pull request by marioflwrs**
I hope I did this right~

### CODE REVIEW

File: Assignment/solution_student/index.html [hunk 1]
```diff
@@ -1,35 +1,66 @@
...
 
-Sample Content
-——-
+<html>
+<head>
+	<title>Home</title>
+	<meta <meta charset="utf-8">
+	<link rel="stylesheet" href="css/main.css">
+</head>
+<h5 id="zz"><strong>About me</strong> | <a href="#">Portfolio</a> | <a href="resume.html">Resume</a></h5>
```
Comment (itsmattsoria) [inline Assignment/solution_student/index.html:10]: Oops! Looks like you forgot an important element here...it comes directly after the `<head>` and it contains aaaaall of the content of the web page...

**Comment by itsmattsoria** [comment]
Not gonna leave a whole lot of feedback just yet, because we'll be going through it in class tomorrow some more, but great job so far! I left one note that I think is important, but great job overall.

File: Assignment/solution_student/index.html (hunk 1 above)
Comment (marioflwrs) [inline Assignment/solution_student/index.html:10]: I think I know what it is....  and It hurts.

the body tag?

**Comment by wawamario** [comment]
TEST  TEST TEST PULL REQUEST TEST

**Pull request closed without merging**